### PR TITLE
resolves positioning on smaller phones

### PR DESCRIPTION
**Pull request by kellgauthier**
## Problem 
Watch button was overlapping Crossroads logo on smaller devices. 
[Rally](https://rally1.rallydev.com/#/detail/defect/443228229228)

## Solution 
Adding media queries and position for smaller devices.

### CODE REVIEW

File: src/components/crds-shared-header/global-nav/global-nav.scss [hunk 1]
```diff
@@ -36,10 +36,14 @@ header {
   display: flex;
 
   a {
-    padding-right: 7px;
     padding-left: 7px;
+    padding-right: 7px;
     &:last-of-type {
       padding-right: 0;
+
+      @media (max-width: $screen-sm) {
```
Comment (silasbaur) [inline src/components/crds-shared-header/global-nav/global-nav.scss:44]: Lets use min-width

File: src/components/crds-shared-header/global-nav/global-nav.scss [hunk 2]
```diff
@@ -49,6 +53,10 @@ header {
   margin: 11px 0 0 -75px;
   position: absolute;
 
+  @media (max-width: $screen-sm) {
```
Comment (silasbaur) [inline src/components/crds-shared-header/global-nav/global-nav.scss:56]: lets try to use just min width here also. Sometimes using a min and max width can complicate things and majority of styles written for crossroads use min-width so lets stay consistent.

**Review by silasbaur — CHANGES_REQUESTED** [review]
Two inline comments and when I goto the smallest device size I am still seeing some issues with the watch button on top of the logo. I would just tighten up the spacing between the icons and see if that helps.
Screen Shot 2020-10-14 at 9 53 48 AM

**Comment by kellgauthier** [comment]
Hi @silasbaur - Thank you for the comments. 
I opted to change all the min-widths to max-widths and here's why: 
In my opinion - _if you have to override this default for smaller screens, use max-width._ 

I tightened up the spacing, and made the logo slightly smaller on the smallest devices (iPhone 4/320px width).

**Review by dholloran-ample — CHANGES_REQUESTED** [review]
@kellgauthier 

1. I have to agree with @silasbaur on the `min-width` vs `max-width` for consistency sake especially since it was already there. 
    - I try to style in this way. All styles outside of the media queries are global/mobile then the media queries overwrite as it gets bigger. I could make a case for `.search-container` since it's only the "desktop" styles (I'm ok either way with that one but rather be consistent)
    - I'm happy to talk through this if you want.
1. It looks like the spacing is better. I have it down to about `305px` before it's touching the logo. The only way I figure we could get it better is if the logo would resize based on screen size. Currently with the logo being absolute (which seems odd but maybe there is a reason) I think it'd be difficult to pull off. I'll leave it up to you if you want to try it.

**Comment by kellgauthier** [comment]
@silasbaur and @dholloran-ample - I did the same thing with this defect. I completely went back and refactored the code utilizing Bootstrap and min-width. I did change the logo size on small screens but I had a thought. 

What if on smaller screens, the logo with the full "Crossroads Church" name, just changed to the logo I attached? 
crossroads-x-logo

Let me know your thoughts and I can do the logo swap on mobile tomorrow if we think it's a good idea.

**Review by silasbaur — CHANGES_REQUESTED** [review]
Changing the way the logo is displayed is a big change and would have to go through a couple higher ups at Crossroads, so, let's just keep the logo how it is.
When I look at your deploy preview the logo is not centered anymore. Make sure you are looking at your deploy previews and looking for things like this. Also it is helpful for reviewers to have the deploy preview link in your PR description.
Screen Shot 2020-10-15 at 9 36 54 AM

**Review by silasbaur — CHANGES_REQUESTED** [review]
This is still not centered. Double check your work
Screen Shot 2020-10-15 at 10 15 10 AM

File: src/components/crds-shared-header/global-nav/global-nav.scss [hunk 3]
```diff
@@ -36,8 +36,14 @@ header {
...
 
   a {
-    padding-right: 7px;
-    padding-left: 7px;
+    padding-left: 2px;
+    padding-right: 0;
+
+    @media (min-width: $screen-sm) {
+      padding-left: 7px;
+      padding-right: 0;
```
Comment (silasbaur) [inline src/components/crds-shared-header/global-nav/global-nav.scss:44]: The padding does not need to change for bigger screens. Set it back to `padding-right: 7px;`

**Review by silasbaur — CHANGES_REQUESTED** [review]
One inline comment and I would remove anything you did to the logo. If you look at the geo-location icon it disappears for screens smaller than 375px because of this issue. I would do the same thing with the watch button and we can bring it up in refinement to see how everyone feels about this decision.

**Review by silasbaur — APPROVED** [review]

**Pull request merged**